src/js/common/utils/accessible-menus.js

```diff
@@ -66,7 +75,7 @@ function isVisible(element) {
 
   // If the element has children, check them; otherwise, naively assume it's visible
   const childrenAreVisible = element.children.length ?
-    Array.from(element.children).some(e => !hiddenDisplays.includes(getComputedStyle(e).display)) :
+    Array.from(element.children).some(e => isScript(e) || !hiddenDisplays.includes(getComputedStyle(e).display)) :
```

Comment:
Not sure who the right person to validate this change is. @cvalarida, maybe?

Reply:
If I'm reading this right, do we potentially have a script tag in the middle of the HTML for our navigation?

Reply:
Yeah the onclick transformer will append a script tag as a child of the element with an inline onclick to make sure it's in the dom before the handler gets step.

Reply:
Also noting that there was an e2e test that failed before this fix. I should probably make sure the unit tests exercise this code path.

Reply:
Yeah, looks good to me. :+1: